giskard/commands/cli_worker.py

```diff
@@ -160,7 +160,7 @@ def _start_command(is_server, url: AnyHttpUrl, api_key, is_daemon, hf_token=None
 
             run_daemon(is_server, url, api_key, hf_token)
         else:
-            if sys.platform == "win32":
+            if sys.platform == "win32" or sys.version_info < (3, 10):
```

Comment:
uvloop didn't work on my my with python 3.9, but it's technically not related to the feature, so can remove for now